hoomd/mpcd/CollisionMethod.cuh

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2009-2025 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file mpcd/CollisionMethod.cuh
+ * \brief Declaration of CUDA kernels for mpcd::CollisionMethod
+ */
+
+#include "hoomd/BoxDim.h"
+#include "hoomd/HOOMDMath.h"
+#include "hoomd/Index1D.h"
+
+namespace hoomd
+    {
+namespace mpcd
+    {
+namespace kernel
```

Comment:
The MPCD GPU code is organized such that all kernel driver functions are in the `gpu` namespace. Then, the kernels are in the `gpu::kernel` namespace. With this organization, you don't need to prefix all your drivers with `gpu_`.
```suggestion
namespace gpu
```